src/test/java/com/rcv/TabulatorTests.java

```diff
@@ -41,7 +41,7 @@
 class TabulatorTests {
 
   // folder where we store test inputs
-  private static final String TEST_ASSET_FOLDER = "src/test/resources/test_data";
+  private static final String TEST_ASSET_FOLDER = "src/test/resources/com/rcv/test_data";
```

Comment:
Extrapolating from our trajectory to date, I predict that our test files are 60 levels deep by the time we officially release this app.